### PR TITLE
Port Arc60 Solder PCB

### PR DESCRIPTION
**Pull request by stanrc85**
Port for Arc60 solder PCB

### CODE REVIEW

File: keyboards/vertex/arc60/keymaps/vial/config.h [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later */
+
+#pragma once
+
+#define VIAL_KEYBOARD_UID {0x8B, 0x13, 0x1F, 0x2D, 0x2A, 0x37, 0x81, 0x46}
+
+#define VIAL_UNLOCK_COMBO_ROWS { 0, 2 }
+#define VIAL_UNLOCK_COMBO_COLS { 0, 13 }
+#define DYNAMIC_KEYMAP_LAYER_COUNT 4
```
Comment (lesshonor) [inline keyboards/vertex/arc60/keymaps/vial/config.h]:
```suggestion
```
This is the default, so you don't really need to mention it.

File: keyboards/vertex/arc60/keymaps/vial/rules.mk [hunk 2]
```diff
@@ -0,0 +1,7 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
```
Comment (lesshonor) [inline keyboards/vertex/arc60/keymaps/vial/rules.mk]:
```suggestion
```
ARM can get buggy with LTO and you probably don't need to be concerned about size.

File: keyboards/vertex/arc60/keymaps/vial/vial.json [hunk 3]
```diff
@@ -0,0 +1,284 @@
+{
+    "name": "arc60",
+    "vendorId": "0x8354",
+    "productId": "0x8370",
+    "lighting": "qmk_rgblight",
+    "matrix": {"rows": 5, "cols": 15},
+  "layouts": {
+  "labels":[
+        "Backspace",
```
Comment (lesshonor) [inline keyboards/vertex/arc60/keymaps/vial/vial.json]:
```suggestion
        "Split Backspace",
```

**Review by lesshonor — APPROVED** [review]
Passes basic sanity checks (board exists, code compiles, no duplicate UID, vial.json loads without error)

File: keyboards/vertex/arc60/keymaps/vial/rules.mk [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+VIAL_ENABLE = yes
+COMBO_ENABLE = no
+KEY_OVERRIDE_ENABLE = no
```
Comment (lesshonor) [inline keyboards/vertex/arc60/keymaps/vial/rules.mk]:
```suggestion
```
Do you really need to disable these?

**Comment by stanrc85** [comment]
All the issues have been addressed, thank you!

**Pull request merged**